patches/server/0013-Paper-Plugins.patch

```diff
@@ -4857,6 +4857,10 @@ index 0000000000000000000000000000000000000000..45bd29b70782e29eb11c36eaca0f940a
 +                .build();
 +        }
 +
++        if (!node.node("commands").virtual()) {
++            org.slf4j.LoggerFactory.getLogger(pluginConfiguration.getName()).warn("Defining commands in paper-plugin.yml is not supported! - Commands will not be available!");
```

Comment:
You could probably link to the docs page (https://docs.papermc.io/paper/dev/getting-started/paper-plugins#commands) that explains what to do instead.